js/notify.js

```diff
@@ -15,7 +15,7 @@ define([
             level: type == 'error' ? 'danger' : type,
             message: message
         }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
+        Log.default.debug('Notify:', {type: type, message: message, container: container});
```

Comment:
The other alternative is to move from AMD to ES6 syntax. Then, the `default` won't be necessary anymore.

Just exporting `error` and `success` won't change the signature so there won't be copatibility issue for AMD module using the notify module.